test/Serializer.js

```diff
@@ -0,0 +1,251 @@
+const chaiAsPromised = require("chai-as-promised");
+const chai = require("chai");
+chai.use(chaiAsPromised);
+const {expect} = chai;
+const dynamoose = require("../lib");
+const {Model} = dynamoose;
+const Serializer = require("../lib/Serializer");
+
+describe("Serializer", () => {
+	let User;
+	beforeEach(() => {
+		User = new Model("User",
+			{id: Number, name: String, email: String, phone: String, passwordHash: String, status: String},
+			{"create": false, "waitForActive": false}
+		);
+	});
+	afterEach(() => {
+		User = null;
+	});
+
+	it("Should verify Model and Document have a serializer instance and expected methods bound", () => {
+		const doc = new User({"id": 1, name: "User"});
+		expect(User.serializer).to.be.an.instanceof(Serializer);
+		expect(doc.serialize).to.be.a("function");
+		expect(User.serializeMany).to.be.a("function");
+	});
+
+	it("Should add some serializers to the Models serializer instance", () => {
+		addSerializers();
```

Comment:
Seems like this happens on almost every test. How do we feel about adding this to a `beforeEach`?

Reply:
Fixed in 8ea90a9c1380a02ea8172a4300abfe9751f3743a